application/src/main/resources/application.yml

```diff
@@ -198,7 +192,11 @@ ipf:
 validation:
   external-terminology:
     enabled: false
-
+    provider:
+      fhir:
+        type: FHIR
+        url: https://r4.ontoserver.csiro.au/fhir/
```

Comment:
I would suggest to remove this url by default

Reply:
I would agree, but the url was present in the code before refactoring. Anyway, the validation is disabled yb default